css/style.css

```diff
@@ -0,0 +1,134 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #f5f5f5;
+  top: 0;
+
+}
+#header {
+  position: sticky;
+  background: rgb(0, 0, 0);
+  height: 100px;
+  top: 0;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;;
+}
+img{
+  width: 100px;
+  margin: 10px 0 0 50px;
+
+}
+#nav-bar{
+  top: 0;
+  margin: 20px 50px 0 0;
+  display: flex;
+  justify-content: space-between;
+  width: max-content;
+  height: 50px;
+}
+#nav-bar a{
+  color: white;
+  text-decoration: none;
+  font-size: 20px;
+  padding: 10px;
+}
+#form-marg{
+  border: solid;
+  height: 200px;
+}
+#form{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
According to [CSS & HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the overuse of IDs. Instead, try to use classes.